src/Drivers/VectoryLinkDriver.php

```diff
@@ -52,29 +53,27 @@ public function message(string $message): string
 
     public function send(): SMSDriverResponseInterface
     {
-        $response = HTTP::get($this->endPoint, $this->headers(), $this->payload());
+        $response = HTTP::post($this->endPoint, $this->headers(), $this->payload());
 
         return new VectoryLinkResponse($response);
     }
 
-    protected function payload(): array
+    protected function payload(): string
     {
-        return
-            [
-                "SMSText" => $this->message,
-                "SMSReceiver" => $this->recipients,
-                "SMSSender" => $this->senderName,
-                'SMSLang' => $this->lang,
-                'UserName' => $this->username,
-                'Password' => $this->password
-            ];
+        return http_build_query([
+            "SMSText" => $this->message,
+            "SMSReceiver" => $this->recipients,
+            "SMSSender" => $this->senderName,
+            'SMSLang' => $this->lang,
+            'UserName' => $this->username,
+            'Password' => $this->password
+        ]);
     }
 
     protected function headers(): array
     {
         return [
-            'Content-Type' => 'text/xml; charset=utf-8',
-            'Content-Length' => 0
+            'Content-Type' => 'application/x-www-form-urlencoded'
```

Comment:
@Mostafa7000 was this header added based on the documentation? I believe the requests are in JSON format so `application/json` would be a more suitable value unless their documentation requires this value.

Reply:
@muhamadhhassan 
Please refer to [the above discussion](https://github.com/RobustaStudio/Resala/pull/18/files#r1713835087)